tcms/kiwi_auth/tests/test_admin.py

```diff
@@ -188,3 +194,26 @@ def test_should_be_able_to_edit_a_non_default_group(self):
             '<input type="text" name="name" value="%s" class="vTextField"'
             ' maxlength="150" required id="id_name">' % self.group.name,
         )
+
+    def test_should_be_allowed_to_create_new_group_with_added_user(self):
+        group_name = "TestGroupName"
+        response = self.client.post(reverse("admin:auth_group_add"), {
+            "name": group_name,
+            "users": [self.tester.id]
+
+        }, follow=True)
+        self.assertContains(response, group_name)
+        self.assertContains(response, "was added successfully.")
+        self.assertIsNotNone(User.objects.filter(
+            groups__name=group_name).filter(pk=self.tester.id).get())
```

Comment:
FYI you can also use the related object manager, e.g. something like `self.tester.groups.filter(name=group_name).exists()`.
-----------------^ Note: I haven't checked what the actual related objects manager name is in this case.